### PR TITLE
Several bugfixes on taxes compares and updates

### PR DESCRIPTION
**Pull request by antespi**
[FIX] Set updated taxes as active
[FIX] Check parent_id field in '_is_different_tax' method
[FIX] PEP8

### CODE REVIEW

File: account_chart_update/wizard/wizard_chart_update.py [hunk 1]
```diff
@@ -560,12 +560,14 @@ def _find_tax_codes(self, mapping_tax_codes):
...
 
-    def _is_different_tax(self, tax, tax_template, mapping_tax_codes,
-                          mapping_accounts):
+    def _is_different_tax(self, tax, tax_template, mapping_taxes,
+                          mapping_tax_codes, mapping_accounts):
         """Check the tax for changes.
         :return: An string that will be empty if no change detected.
         """
         notes = ""
+        if tax.active == False:
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: Better `if not tax.active:`

File: account_chart_update/wizard/wizard_chart_update.py [hunk 2]
```diff
@@ -560,12 +560,14 @@ def _find_tax_codes(self, mapping_tax_codes):
...
+            notes += _("Tax is disabled.\n")
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py:572]: Add this string to the translation files.

File: account_chart_update/wizard/wizard_chart_update.py [hunk 3]
```diff
@@ -582,6 +584,9 @@ def _is_different_tax(self, tax, tax_template, mapping_tax_codes,
             notes += _("The domain field is different.\n")
         if tax.child_depend != tax_template.child_depend:
             notes += _("The child depend field is different.\n")
+        if tax.parent_id != self.map_tax_template(tax_template.parent_id,
+                                                  mapping_taxes):
+            notes += _("The parent_id field is different.\n")
```
Comment (pedrobaeza) [inline account_chart_update/wizard/wizard_chart_update.py]: Add this string to the translations

**Comment by antespi** [comment]
I've also update .pot file, no updated since v5. But I can only update  Spanish translation.

**Comment by pedrobaeza** [comment]
:+1:

**Comment by Endika** [comment]
:+1:

**Comment by pedrobaeza** [comment]
Runbot warning unrelated, so I merge

**Pull request merged**